src/hierarchies/base_hierarchy.h

```diff
@@ -154,8 +155,8 @@ void BaseHierarchy<Derived, State, Hyperparams, Prior>::add_datum(
   assert(cluster_data_idx.find(id) == cluster_data_idx.end());
   card += 1;
   log_card = std::log(card);
-  static_cast<Derived *>(this)->update_summary_statistics(datum, covariate,
-                                                          true);
+  static_cast<Derived *>(this)->get_summary_statistics_update(datum, covariate,
```

Comment:
This is bad naming IMO, `get_summary_statistics_update` does not return anything right?

Reply:
True, but nothing better came to mind, so I just followed the pattern from the lpdf functions. Any suggestions?

Reply:
since `update_summary_statistics()` is called only by `BaseHierarchy::add/remove_datum()`, we could rename

`update_summary_statistics` --> `ss_update`

`get_summary_statistics_update ` --> `update_summary_statistics`

WDYT?

Reply:
Do you mean the other way round? We should give a meaningful name to the function that the user has to define, which is `update_summary_statistics()`.

Reply:
I like the pattern, though. Maybe
```
update_summary_statistics -> summary_statistics_update
get_summary_statistics_update -- do/perform_update_summary_statistics
```
?

Reply:
> Do you mean the other way round? We should give a meaningful name to the function that the user has to define, which is `update_summary_statistics()`.

Yes of course, the one defined by the user should have the better name

`perform_update_summary_statistics` that's a 33 chars name and we work with 79 chars lines... I think that `update_ss()` is good enough